internal/tools/policyenginesimulator/README.md

```diff
@@ -0,0 +1,217 @@
+# Policy Engine Simulator
+
+This tool provides a means to test out security policies. Usage:
+
+```
+  -commands string
+        commands JSON
+  -data string
+        initial data state
+  -log string
+        log path
+  -logLevel string
+        None|Info|Results|Metadata (default "Info")
+  -policy string
+        policy Rego
+```
+
+## Getting started
+
+From the tool directory run:
+
+   go run . -policy [samples/simple_framework/policy.rego](samples/simple_framework/policy.rego) -commands [samples/simple_framework/commands.json](samples/simple_framework/commands.json)
+
+This will load the authored policy and then simulate the enforcement behavior
+for the provided commands.
+
+This policy uses the framework, however, the simulator also handles completely
+custom policies:
+
+   go run . -policy [samples/simple_custom/policy.rego](samples/simple_framework/policy.rego) -commands [samples/simple_custom/commands.json](samples/simple_custom/commands.json)
+
+## Commands
+
+Consists of a sequential list of commands that will be issued for enforcement to
+the policy. These commands take the form of JSON, *e.g.*:
+
+``` json
+[
+    {
+        "name": "load_fragment",
+        "input": {
+            "issuer": "did:web:contoso.github.io",
+            "feed": "contoso.azurecr.io/custom",
+            "namespace": "custom",
+            "local_path": "custom.rego"
+        }
+    },
+    {
+        "name": "mount_device",
+        "input": {
+            "target": "/mnt/layer0",
+            "deviceHash": "16b514057a06ad665f92c02863aca074fd5976c755d26bff16365299169e8415"
+        }
+    },
+    {
+        "name": "mount_overlay",
+        "input": {
+            "target": "/mnt/overlay0",
+            "containerID": "container0",
+            "layerPaths": [
+                "/mnt/layer0"
+            ]
+        }
+    },
+    {
+        "name": "create_container",
+        "input": {
+            "containerID": "container0",
+            "argList": [
+                "/pause"
+            ],
+            "envList": [
+                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+                "TERM=xterm"
+            ],
+            "mounts": [],
+            "workingDir": "/",
+            "sandboxDir": "/sandbox",
+            "hugePagesDir": "/hugepages"
+        }
+    }
+]
+```
+
+Each command has a name (corresponding to the API enforcement point) and then
```

Comment:
Is there a place where you document the API enforcement points (what they are and what they are used for)? If so, I would add a link to that here, to make it easier to navigate the repo.

Reply:
I've added a link.